### ISSUE TITLE
Exeggutor sprite overlaps the sprite of Cubone

### ISSUE THREAD

**Issue by ajimenezml** [AHAAAAAAA/PokemonGo-Map#2218]
It is happening on both master and develop branches.

![Image](https://s31.postimg.org/o9jccyirf/Image.png)

## Expected Behavior
Exeggutor sprite should not overlap the sprite of Cubone.

## Current Behavior
Exeggutor sprite is overlapping the sprite of Cubone.

## Possible Solution
Make the sprite of Exeggutor one pixel less in width.

## Steps to Reproduce (for bugs)
Search in the map a place where Cubone appears (for example, "Madrid, Spain").

## Context
It is only an aesthetic error, nothing important.

## Your Environment
* Version used: Both master and develop branches
* Environment name and version (e.g. Python 2.7): Python 2.7.12
* Operating System and version (desktop or mobile): Ubuntu Server 16.04.1 LTS, Desktop

**Comment by JasperG** [comment]
Then, fix it. Move the icon one or two px to the left, and push to github.

**Comment by medghaim** [comment]
Nice catch!

**Comment by invisiblek** [comment]
While you're at it, can you ensure others are not like this?

Also is this happening in both icons-sprite.png and icons-large-sprite.png?

**Comment by invisiblek** [comment]
weird, it doesn't do it for me in gimp (icons-sprite.png)

**Comment by ajimenezml** [comment]
Yes, it is happening in both icons-sprite.png and icons-large-sprite.png, and in icons-large-sprite.png Graveler is overlapping Geodude and Moltres is overlapping Dratini also.

If necessary, I can correct the image files with Photoshop, since I have the grid prepared to detect these overlaps.

**Comment by invisiblek** [comment]
That would be fantastic!

**Comment by Kevinclon** [comment]
was this done? if not ill go check all the sprites

**Comment by ajimenezml** [comment]
I am at work right now, as I arrive to my home (in a couple of hours) I will upload both images corrected without any Pokémon overlapping the sprite of another one.

**Comment by ajimenezml** [comment]
Here they are:

- [icons-sprite.png](http://i.imgur.com/gu9w2ul.png)
- [icons-large-sprite.png](http://i.imgur.com/hrLtgiU.png)

Can you @invisiblek upload the commit or should I create a pull request?

**Comment by invisiblek** [comment]
If you want to make a pull request that'd be cool. Then it would have your authorship too, so when this is broken 👅 we know who to track down

**Comment by ajimenezml** [comment]
Well, I just created the pull request, and then I thought about checking all the Pokémon on the map and I found this...

![Image](http://i.imgur.com/yh4IN7P.png)

They are the corrected sprites, but with a pink background and a border of 1 px around each Pokémon.

As can be seen, apparently Google Maps is cutting the sprites without maintaining consistency across all the zoom levels, sometimes cutting more than it should and other less. This is causing in the high resolution icons that sometimes a Pokémon is shown cut and sometimes a piece of a Pokémon is appearing in the sprite of another Pokémon.

In my pull request I corrected the sprites to fit their grid, but now we must find a solution to this new problem...
